OrchardCore/Init-OrchardCoreSolution/Init-OrchardCoreSolution.psm1

```diff
@@ -0,0 +1,27 @@
+﻿<#
+.Synopsis
+    Initializes an Orchard Core solution for a git repository. Deprecated, use Initialize-OrchardCoreSolution instead.
+#>
+
+function Init-OrchardCoreSolution
```

Comment:
I wanted to suggest adding `Init-OrchardCoreSolution` as an alias to `Initialize-OrchardCoreSolution`, but it's already there. I added it almost exactly a year ago and the comments referencing it are older than that, so that shouldn't be a problem anymore - at least for Windows PS.

However, PS Core behaves differently and doesn't export aliases by default (for performance reasons: https://github.com/PowerShell/PowerShell/issues/9379, whatever that means), so I experimented with manifests a bit (it was a bit of a rabbit hole, because they behave differently compared to Windows PS) and managed to set one up that works seamlessly. It is the absolute minimal manifest to make this work in this specific, but in reality, it should be set up to include e.g., all the cmdlets under the OrchardCore folder (so OrchardCore is the module and it has multiple cmdlets). I'll experiment with that a bit and then create an issue, because it would allow us to publish these like NuGet packages.

Can you please verify? If it works, then we don't need a separate cmdlet.

Reply:
I see, interesting! I tried it and it works both with pwsh and Windows PowerShell. I also got rid of the other proxy script.

Reply:
Deleting code while keeping functionality is super satisfying! :)

Reply:
Yep!